# Define the goal
geometry_msgs/PoseStamped       target_pose
bool                            turn_before_nav
---
# Define the result
string trace
---
# Define a feedback message
